index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <!-- Header section -->
+    <header class="header">
+      <nav class="nav-bar">
+        <h3>Ha Ell</h3>
+        <img src="./image_icons/Icon-Menu.svg" alt="bar icon"/>
+      </nav>
+    </header>
+    <!-- End of Header section -->
+    <!-- Headline section -->
+    <section class="headline-section">
+      <div class="container">
+        <div class="headline-title">
+          <h1>hey there.</h1>
+          <h1>I am Hamza</h1>
+        </div>
+        <div class="headline-content">
+          <h4 class="content-title">I am a software developer student</h4>
+          <p class="body">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+
+          <div class="social-icon">
+            <div class="social-item">
+              <a href="#" target="_blank"><img src="./image_icons/Icon-GitHub.svg" alt="github"/></a>
+            </div>
+            <div class="social-item">
+              <a href="#" target="_blank"><img src="./image_icons/Icon-Linkedin.svg" alt="Linkedin"/></a>
+            </div>
+            <div class="social-item">
+              <a href="#" target="_blank"><img src="./image_icons/Icon-Angelist.svg" alt="Angelist"/></a>
+            </div>
+            <div class="social-item">
+              <a href="#" target="_blank"><img src="./image_icons/Icon-Twitter.svg" alt="Twitter"/></a>
+            </div>
```

Comment:
- [ ] As pointed out in the screenshot below, all these icons should be in a `<ul>` tag and every single icon wrapped in an `<li>` tag.
 
![icons](https://user-images.githubusercontent.com/45002654/124990779-7b40a280-e049-11eb-9e6e-699115ddda04.PNG)